platform/android/display_server_android.h

```diff
@@ -70,6 +70,28 @@ class DisplayServerAndroid : public DisplayServer {
 
 	int buttons_state;
 
+	// https://developer.android.com/reference/android/view/PointerIcon
+	// couldn't found a way to import that from ndk
```

Comment:
Can you update this documentation. Should be something like `Mapping from Android PointerIcon types to CursorShape values`.